File: Backend/The-Coffe-Palace-Feign-Clien/src/main/java/com/feign_client/data/model/Reserva.java
package com.feign_client.data.model;

import java.time.LocalDate;
import java.time.LocalTime;



import lombok.Data;

@Data
public class Reserva {
	
		private Long idRes;
	    private Local local;
	    private String nomRes;
	    private String apeRes;
	    private String emailRes;
	    private String telRes;
	    private LocalDate fechaReserva;
	    private LocalTime horaReserva;
	    private int comensales;
	    private String enable = "S";

}

File: Frontend/src/app/modelos/reserva.ts
import { Local } from "./local";

export class Reserva {
  idRes: number;
  local: Local;
  nomRes: string;
  apeRes: string;
  emailRes: string;
  telRes: string;
  fechaReserva: Date;
  horaReserva: { hours: number, minutes: number };
  comensales: number;
  enable: string;

  constructor() {
    this.idRes = 0;
    this.local = new Local();
    this.nomRes = "";
    this.apeRes = "";
    this.emailRes = "";
    this.telRes = "";
    this.fechaReserva = new Date();
    this.horaReserva = { hours: 0, minutes: 0 }; 
    this.comensales = 0;
    this.enable = "S";
  }
}